src/sql/src/plan/func.rs

```diff
@@ -902,6 +902,11 @@ lazy_static! {
                     Ok(ScalarExpr::literal_null(ScalarType::String))
                 })
             },
+            "pg_encoding_to_char" => Scalar {
```

Comment:
Can you add the comment on the test to the function definition instead? One of the odder functions I've seen!